tests/misc/sys_atexit.py

```diff
@@ -1,21 +0,0 @@
-# test sys.atexit() function
-
-import sys
-
```

Comment:
Can you rewrite this to work with the atexit module? It'd be good to have tests for this since it doesn't require hardware to test.

Reply:
Done.